tuf/api/metadata.py

```diff
@@ -407,8 +399,9 @@ def bump_version(self) -> None:
 
 class Key:
     """A container class representing the public portion of a Key.
+
     Please note that "Key" instances are not semanticly validated during
-    initialization. We consider this as responsibility of securesystemslib.
+    initialization: this only happens at signature verification time.
```

Comment:
Do we plan to use the `Key` class besides signature verification?
If no other cases appear in mind then okay, let's be more precise, but if there are other cases we envision it could be better to be a little more abstract.

Reply:
I can't think of anything else we would want to expose... we probably want to add a `Key.from_securesystemslib_key()` still but even there we likely will just assign the attributes and that's it